packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/configuration/SiriusWebMessageServiceConfiguration.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
```

Comment:
We already have a message service in Sirius Web used by multiple services of `sirius-web-application`, do we really need a second one?